source/doc-pages/introduction.md

```diff
@@ -1,35 +1,62 @@
-Ruby Object Mapper (ROM) is a Ruby library with the goal to provide powerful object
-mapping capabilities without limiting the full power of your datastore. ROM is based
-on several concepts and decisions that differentiate it from your normal Ruby ORM:
+Ruby Object Mapper (ROM) is a Ruby persistence library with the goal to provide 
```

Comment:
I like your suggestion @maetl - I also wonder if we should just use the word `database`. Maybe `datastore` is confusing for some people, hard to say.